doc/docusaurus/docs/auction-smart-contract/end-to-end/getting-funds.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 10
+---
+
+# Getting Funds from the Faucet
+
+Next, we'll need to fund the wallet of each participant (seller, bidder1 and bidder2), in order to cover transaction fees and place bids.
+We can get funds from Cardano's [testnet faucet](https://docs.cardano.org/cardano-testnets/tools/faucet/).
+
+To request funds, enter the seller's address into the address field and click "request funds."
+This will deposit 10,000 (test) ADA into the seller's wallet.
+
+Since the faucet limits how frequently you can request funds, and 10,000 ADA is more than sufficient for this example, we'll share the 10,000 ADA among the seller, bidder1, and bidder2.
+To do so, create a file named `off-chain/send-lovelace.mjs` with the following content:
+
+<LiteralInclude file="send-lovelace.mjs" language="javascript" title="send-lovelace.mjs" />
+
+Substitute your Blockfrost project ID for `Replace with Blockfrost Project ID`.
+
+This Javascript module builds and submits a transaction that sends 1 billion Lovelace (equivalent to 1000 Ada) from the seller's wallet to the specified recipient.
+Run the following commands:
+
+```
+node send-lovelace.mjs bidder1
+node send-lovelace.mjs bidder2
```

Comment:
```suggestion
pushd off-chain
node send-lovelace.mjs bidder1
node send-lovelace.mjs bidder2
popd
```

Reply:
previous commands assume that current dir is one level above the `off-chain` dir, but this command fails when executed not from the `off-chain` dir.

Reply:
> previous commands assume that current dir is one level above the off-chain dir

Are you referring to "To do so, create a file named `off-chain/send-lovelace.mjs`"? I can change it to "To do so, create a file named `send-lovelace.mjs` in `off-chain`". The previous task was to generate keys, and there's no need to leave the `off-chain` dir.